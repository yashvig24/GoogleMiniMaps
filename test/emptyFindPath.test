# Tests find path on empty graph.

# Create a graph
CreateGraph empty_graph

# Find minimum-cost path on non-existing nodes
FindPath empty_graph a b