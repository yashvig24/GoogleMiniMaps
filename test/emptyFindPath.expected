# Tests find path on empty graph.

# Create a graph
created graph empty_graph

# Find minimum-cost path on non-existing nodes
unknown character a
unknown character b